lib/mongo/collection.rb

```diff
@@ -107,12 +107,24 @@ def ==(other)
     # @option options [ Hash ] :time_series Create a time-series collection.
     #   See https://mongodb.com/docs/manual/core/timeseries-collections/ for more
     #   information about time-series collection.
+    # @option options [ Hash ] :read_concern The read concern options hash,
+    #   with the following optional keys:
+    #   - *:level* -- the read preference level as a symbol; valid values
+    #      are *:local*, *:majority*, and *:snapshot*
+    # @option options [ Hash ] :read The read preference options.
+    #   The hash may have the following items:
+    #   - *:mode* -- read preference specified as a symbol; valid values are
+    #     *:primary*, *:primary_preferred*, *:secondary*, *:secondary_preferred*
+    #     and *:nearest*.
+    #   - *:tag_sets* -- an array of hashes.
+    #   - *:local_threshold*.
     # @option options [ Integer ] :expire_after Number indicating
     #   after how many seconds old time-series data should be deleted.
     # @options clustered_index [ Hash ] :clustered_index Create a clustered index.
     #   This option specifies how this collection should be clustered on _id.
     #   See https://www.mongodb.com/docs/v5.3/reference/method/db.createCollection/#std-label-db.createCollection.clusteredIndex
     #   for more information about this option.
+    # @option options [ Hash ] :collation The collation to use.
     #
     # @since 2.0.0
     def initialize(database, name, options = {})
```

Comment:
`:view_on` doesn't appear to be documented (https://github.com/mongodb/mongo-ruby-driver/blob/85ac96b1d282d59ab9b7dea235d6e32d3f9e325f/lib/mongo/collection.rb#L75). Might as well add that too.

Reply:
Note it doesn't appear `:pipeline`, which the [`create` command](https://www.mongodb.com/docs/manual/reference/command/create/) would use to filter the view is defined here.

Reply:
done!